pdns/recursordist/rec-main.cc

```diff
@@ -1687,6 +1687,8 @@ static int initSyncRes(Logr::log_t log)
   SyncRes::s_ecscachelimitttl = ::arg().asNum("ecs-cache-limit-ttl");
 
   SyncRes::s_qnameminimization = ::arg().mustDo("qname-minimization");
+  SyncRes::s_minimize_one_lab = ::arg().asNum("qname-minimize-one-label");
```

Comment:
It's not obvious to me without the context that `lab` in `minimize_one_lab`  means `label`, perhaps:
```suggestion
  SyncRes::s_minimize_one_label = ::arg().asNum("qname-minimize-one-label");
```

Reply:
Right. The name comes from the RFC, but full `label`` is better.